code/modules/cargo/packs/science.dm

```diff
@@ -10,6 +10,46 @@
 	group = "Science"
 	crate_type = /obj/structure/closet/crate/science
 
+/* For later
+/datum/supply_pack/science/monkey
+	name = "Ape Cube Crate"
+	desc = "Pss what a new test subject with supper strangth, speed, and love for bananas all at the same time? Say no more... Contains a single ape cube. Dont add water!"
+	contraband = TRUE
+	cost = 2500
+	contains = list (/obj/item/reagent_containers/food/snacks/monkeycube/ape)
+	crate_name = "ape cube crate"
+*/
+
+/datum/supply_pack/science/aliens
+	name = "Advanced Alien Alloy Crate Crate"
+	desc = "Hello brothers from the stars!!! Are fellow brethren have made contact at long last and gave us gifts man! They really did build the prymi- Connection Error- Bro will send you a sheet of advanced alien alloy "
+	cost = 9000
```

Comment:
Nope, it unlocks just able to make more alien alloy you need 2 alien tools/stuff to really make the adv tools/other things
This might be able to unlock alien surgery tho but im thinking that needs alien bio?

Reply:
that got changed, i'd recommend making the alien alloy crate REALLY expensive (along the lines of multiple tens of thousands)

Reply:
Kappa is correct changing it accordingly